### PR TITLE
DOCS-8711 Static Analysis Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Add a tile for PHP in the language selector section of the Static Analysis product landing page.

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

https://docs-staging.datadoghq.com/alai97/add-static-analysis-language-tile/code_analysis/static_analysis

### CODE REVIEW

File: layouts/partials/code_analysis/languages-getting-started.html [hunk 1]
```diff
@@ -1,70 +1,80 @@
...
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/javascript_large.png" "class" "img-fluid" "alt" "javascript" "width" "600") }}
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/javascript_large.png" "class" "img-fluid" "alt" "javascript" "width" "50") }}
           </div>
         </a>
       </div>
       <div class="col">
         <a class="card h-100" href="/code_analysis/static_analysis_rules?languages=TypeScript">
           <div class="card-body text-center py-2 px-1 d-flex justify-content-center align-items-center">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/typescript_large.svg" "class" "img-fluid" "alt" "typescript" "width" "1000") }}
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/typescript_large.svg" "class" "img-fluid" "alt" "typescript" "width" "50") }}
```
Comment (alai97) [inline layouts/partials/code_analysis/languages-getting-started.html:22]:
```suggestion
            {{ partial "img.html" (dict "root" . "src" "integrations_logos/typescript_large.svg" "class" "img-fluid" "alt" "typescript" "width" "100") }}
```

File: layouts/partials/code_analysis/languages-getting-started.html [hunk 2]
```diff
@@ -1,70 +1,80 @@
...
           <div class="card-body text-center py-2 px-1 d-flex justify-content-center align-items-center">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby_avatar.svg" "class" "img-fluid" "alt" "ruby" "width" "200") }}
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby_avatar.svg" "class" "img-fluid" "alt" "ruby" "width" "50") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/code_analysis/static_analysis_rules?languages=PHP">
+          <div class="card-body text-center py-2 px-1 d-flex justify-content-center align-items-center">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/php_opcache.png" "class" "img-fluid" "alt" "php" "width" "50") }}
```
Comment (alai97) [inline layouts/partials/code_analysis/languages-getting-started.html:60]:
```suggestion
            {{ partial "img.html" (dict "root" . "src" "integrations_logos/php_opcache.png" "class" "img-fluid" "alt" "php" "width" "80") }}
```

File: layouts/partials/code_analysis/languages-getting-started.html [hunk 3]
```diff
@@ -1,70 +1,80 @@
...
         </a>
       </div>
+    </div>
+  </br>
+    <div class="row justify-content-center"> 
       <div class="col">
         <a class="card h-100" href="/code_analysis/static_analysis_rules?languages=Go">
           <div class="card-body text-center py-2 px-1 d-flex justify-content-center align-items-center">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/golang-avatar.png" "class" "img-fluid" "alt" "go" "width" "200") }}
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/golang-avatar.png" "class" "img-fluid" "alt" "go" "width" "50") }}
```
Comment (alai97) [inline layouts/partials/code_analysis/languages-getting-started.html:46]:
```suggestion
            {{ partial "img.html" (dict "root" . "src" "integrations_logos/golang-avatar.png" "class" "img-fluid" "alt" "go" "width" "60") }}
```

File: layouts/partials/code_analysis/languages-getting-started.html [hunk 4]
```diff
@@ -1,70 +1,80 @@
...
           </div>
         </a>
       </div>
       <div class="col">
         <a class="card h-100" href="/code_analysis/static_analysis_rules?languages=Ruby">
           <div class="card-body text-center py-2 px-1 d-flex justify-content-center align-items-center">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby_avatar.svg" "class" "img-fluid" "alt" "ruby" "width" "200") }}
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby_avatar.svg" "class" "img-fluid" "alt" "ruby" "width" "50") }}
```
Comment (alai97) [inline layouts/partials/code_analysis/languages-getting-started.html:53]:
```suggestion
            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby_avatar.svg" "class" "img-fluid" "alt" "ruby" "width" "45") }}
```

**Review by devindford — APPROVED** [review]
LGTM, thanks for making those tweaks

**Pull request merged**